External/INCHI-API/inchi.cpp

```diff
@@ -1964,6 +1966,13 @@ std::string MolToInchi(const ROMol &mol, ExtraInchiReturnValues &rv,
 
     // neighbor
     unsigned int idx = inchiAtoms[atomIndex1].num_bonds;
+    // The InChI code has a max number of neighbors allowed:
+    if (idx >= MAXVAL) {
+      BOOST_LOG(rdErrorLog)
+          << " atom " << atomIndex1 << " has too many bonds: " << idx
+          << ". The InChI library supports at most " << MAXVAL - 1 << std::endl;
```

Comment:
Why `MAXVAL - 1` ?  the neighbors array is defined as `AT_NUM  neighbor[MAXVAL];`, so there should be space for `MAXVAL` neighbors (no null terminators here).